peribolos/knative-sandbox.yaml

```diff
@@ -474,6 +474,19 @@ orgs:
             - chizhg
             - chaodaiG
             privacy: closed
+      GitHub Actions Effort:
+        description: Grants admin rights to github actions related repositories.
+        privacy: closed
+        repos:
+          .github: admin
+          downstream-test-go: admin
+        teams:
```

Comment:
I don't think you need a sub-team here, just include `members` in the upper "effort" directly.

Reply:
ok I think I did, can you double check that?